CHANGES.rst

```diff
@@ -59,6 +59,8 @@ astropy.modeling
   See modeling documentation for details.
   `<https://docs.astropy.org/en/v4.0/modeling/changes_for_4.html>`_ . [#8769]
 
+- Significant reorganization of the documentation.
```

Comment:
You need to add the PR number at the end of the change entry, like [#9078]

Reply:
Forgot that.  Will do so.